src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/main.yml

```diff
@@ -69,8 +69,12 @@ sylius_backend_promotion_action:
     prefix: /promotion-actions
 
 sylius_backend_tax_category:
-    resource: @SyliusWebBundle/Resources/config/routing/backend/tax_category.yml
-    prefix: /tax-categories
+    resource: |
```

Comment:
Why raw YAML ? 
You can't get object here: https://github.com/Sylius/Sylius/pull/3776/files#diff-7ca9ea47a8748302cf1a4e9ceb8faa40R51 ?